src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/commerce/list/.content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
```

Comment:
As the CIF list is a dropin replacement, wouldn't it make more sense to change the resource type of the normal list component when cif is enabled?

This is the way it was done for button https://github.com/adobe/aem-project-archetype/blob/d322424553beeefd1c689cfbeda9608f36509275/src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/button/.content.xml